Cargo.toml

```diff
@@ -13,5 +13,13 @@ log = "0.4"
 async-trait = "0.1"
 eyre = "0.6"
 color-eyre = "0.6"
+aws-config = "0.51.0"
```

Comment:
All versions should be `major.minor` UNLESS we are explicitly trying to lock one of the versions or if their docs say to just use `major`. Please update or comment otherwise.